### ISSUE TITLE
Error while installing in eclipse

### ISSUE THREAD

**Issue by soheilrahsaz** [gwt-plugins/gwt-eclipse-plugin#479]
After following the steps in this link
https://gwt-plugins.github.io/documentation/gwt-eclipse-plugin/Download.html
on a fresh downloaded eclipse from this link
https://www.eclipse.org/downloads/download.php?file=/technology/epp/downloads/release/2023-12/R/eclipse-jee-2023-12-R-win32-x86_64.zip

it shows this error:
```
Cannot complete the install because one or more required items could not be found.
  Software being installed: GWT Eclipse Plugin 4.0.0.202302020242 (com.gwtplugins.eclipse.suite.v3.feature.feature.group 4.0.0.202302020242)
  Missing requirement: GWT Eclipse Plugin 4.0.0.202302020242 (com.gwtplugins.gdt.eclipse.suite 4.0.0.202302020242) requires 'osgi.bundle; javax.inject 1.0.0' but it could not be found
  Cannot satisfy dependency:
    From: GWT Eclipse Plugin 4.0.0.202302020242 (com.gwtplugins.eclipse.suite.v3.feature.feature.group 4.0.0.202302020242)
    To: org.eclipse.equinox.p2.iu; com.gwtplugins.gdt.eclipse.suite [4.0.0.202302020242,4.0.0.202302020242]
```

**Comment by niloc132** [comment]
Duplicate of https://github.com/gwt-plugins/gwt-eclipse-plugin/issues/474, closing.

We have a fix merged, you can download nightly builds from https://github.com/gwt-plugins/gwt-eclipse-plugin/actions/runs/7872612432 (under artifacts, download the repository.zip). Updated release is waiting on https://github.com/gwt-plugins/gwt-eclipse-plugin/issues/478.
